examples/cert.csr

```diff
@@ -0,0 +1,17 @@
+-----BEGIN CERTIFICATE REQUEST-----
```

Comment:
We shouldn't commit this into the repository as it's not needed and it can be easily regenerated if needed.

Reply:
Fixed, and updated gen-certs.sh to follow this policy.